carma_cooperative_perception/src/msg_conversion.cpp

```diff
@@ -267,6 +269,23 @@ auto to_detection_list_msg(
       ref_pos_map.easting + pos_offset.offset_x, ref_pos_map.northing + pos_offset.offset_y,
       ref_pos_map.elevation + pos_offset.offset_z.value_or(units::length::meter_t{0.0})});
 
+    // Pose covariance is x, y, z, roll, pitch, yaw
+    try {
+      detection.pose.covariance.at(0) =
```

Comment:
Is covariance a flattened out 3X3 matrix?

Reply:
Flattened out 6x6 matrix. I updated the comment to be more descriptive.